test/python_kc_mwm/src/environmentModule.py

```diff
@@ -4,29 +4,26 @@
 # (See accompanying LICENSE file or at
 #  https://opensource.org/licenses/MIT)
 
+from copy import deepcopy
+from tqdm import tqdm
+from cmath import inf
+import pickle as pkl
 import numpy as np
-import math
+import statistics
 import itertools
 import argparse
+import shutil
 import time
-import pickle as pkl
+import math
 import pdb
 import os
-import shutil
-import statistics
 import re
-from copy import deepcopy
-from cmath import inf
-from itertools import combinations, permutations
-from tqdm import tqdm
-from ctypes import Structure, c_double
-from statistics import mean
-# import bitstring
 
 np.set_printoptions(linewidth=np.inf)
 np.set_printoptions(threshold=np.inf)
+
 # np_impa_lib = np.float32
 np_impa_lib = np.float64
-
 zero_value = np_impa_lib(0)
-# np.random.seed(17)
+
+# np.random.seed(17)
```

Comment:
To resolve the error, we need to add:

from itertools import combinations, permutations

Reply:
With regards to your email: 

I've added back this import statement, along with the one in the previous comment. I've tested the Header-only demos, pure python code, python wrapper code, and each of the unit test scripts and everything looks okay on my end. 

Interestingly enough I was not getting errors prior to adding back these imports, so perhaps you want to double check once more on your end that everything looks okay still

Reply:
Now things are working well on my end. Whenever I make changes to the code locally, I run 
'python3 -m pip install . -v'